service/model_manager/model_list.py

```diff
@@ -0,0 +1,47 @@
+import os
+import folder_paths
+import server
+from aiohttp import web
+import hashlib
+
+comfy_path = os.path.dirname(folder_paths.__file__)
+
+model_root_dir = os.path.join(comfy_path, 'models')
+
+
+# save file hash dict
+file_hash_dict = {}
+
+@server.PromptServer.instance.routes.get("/model_manager/get_model_list")
+def get_model_list(request):
+    file_list = []
+    folders = [folder for folder in os.listdir(model_root_dir) if os.path.isdir(os.path.join(model_root_dir, folder))]
+    for folder in folders:
```

Comment:
can you use `get_filename_list()` in folder_paths like i mentioned? this will give you cache and it is doing recursive search (subfolders), once you have the array of files, use get_full_path() in folder_paths to get the path

we should try to reuse the utils in comfyui's backend infra as much as possible